cfpb/consumerfinance.gov
# Remove elasticsearch2 index from docker-stack.yml

Now that the elasticsearch2 container has been removed from the docker-stack there is no need to run the `update_index` management command.

## Removals

- `./cfgov/manage.py update_index` from `docker-stack.yml`


## How to test this PR

1. Confirm that no elasticsearch2 indexes are updated
2. Verify that elasticsearch7 indexes are not impacted by this change

## Checklist

  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance
